core/src/jsMain/kotlin/dev/fritz2/validation/validation.kt

```diff
@@ -168,12 +168,14 @@ fun <D, M> WithJob.storeOf(
     ValidatingStore(initialData, validation, Unit, job, validateAfterUpdate = true, id)
 
 /**
- * Finds all corresponding [ValidationMessage]s to this [Store].
+ * Finds all corresponding [ValidationMessage]s to this [Store] which satisfy the [filterPredicate]-expression.
  *
  * Be aware that the  filtering is based upon the correct usage of [Store.path]'s field. This can be reliably achieved
  * by using [dev.fritz2.core.Inspector]s and their mappings for creating the correct path values.
+ *
+ * @param filterPredicate expression to filter messages.
  */
-fun <M : ValidationMessage> Store<*>.messages(): Flow<List<M>>? =
+fun <M : ValidationMessage> Store<*>.messages(filterPredicate: (M) -> Boolean): Flow<List<M>>? =
```

Comment:
why not add a default parameter?

Reply:
In order to "remove" the parameterless variant?
The API Doc is much more expressive this way - if we chose yout approach, one would have to read until the default-value and thus its behaviour is explained...

Reply:
Ok, valid point